.travis.yml

```diff
@@ -41,15 +41,19 @@ addons:
   apt:
     update: true
 
-before_script:
+install:
   - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
   - sudo apt-get update -qq
   - sudo apt-get install --yes -qq gcc jq
   - sudo apt-get install --yes -qq libudev-dev
   - sudo apt-get install --yes -qq zip
```

Comment:
Why do local PV provisioner require these dependencies?

Reply:
only zip is required for building. Will remove the rest.